react/week3/todos/src/components/counter.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+class Counter extends React.Component {
+    constructor(props) {
+      super(props)
+      this.state = {
+        value: 0
+      }
+      this.timer = this.timer.bind(this)
+    }
+  
+    componentDidMount() {
+      this.counter = setInterval(this.timer, 1000);
+    }
+  
+    componentWillUnmount() {
+      clearInterval(this.counter);
+    }
+  
+    timer() {
```

Comment:
if you use arrow functions, you can avoid the need to use `.bind` in the constructor
```suggestion
    timer = () => {
```